docs/integrations/netskope.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Netskope Integration
+category: 62cdf6722c2c1602a4b69643
+hidden: false
+---
+
+# Netskope Integration
+
+## Overview
+
+Phylum can be integrated with [Netskope](https://www.netskope.com) to incorporate Phylum's curated list of malicious open-source package hashes into a Netskope security policy.
+
+## Getting Started
+
+Netskope users who purchase a Phylum threat feed subscription can follow the instructions below to subscribe to Phylum's curated malicious package hash list distribution. This list will be automatically reflected in your Netskope `File Hash List` on a regular cadence. There are three separate pieces of information needed to complete the subscription process:
+
+1. A Netskope v1 API Token for your tenant
+2. Your Netskope tenant name
+3. A Netskope `File Hash List` name
+
+### Obtain a Netskope v1 API Token
+
+1. Log in to your Netskope admin dashboard and then click on `Settings`.
+2. In `Settings` click on `Tools`.
+3. In `Tools` click on `Rest API v1`.
+4. Grab the Token value from Token section.
+
+### Obtain Your Netskope Tenant Name
+
+1. If you do not have a Netskope Tenant configured, follow [these directions](https://docs.netskope.com/en/netskope-help/integrations-439794/netskope-cloud-exchange/get-started-with-cloud-exchange/configure-netskope-tenants/) provided by Netskope. Upon completing this configuration, the name you chose will be the Tenant Name.
+2. If you already have a tenant configured, your Netskope Tenant name is simply the subdomain of your dashboard URL. For example when you log into your Netskope dashboard, look at the URL. It should be of the form `https://<tenant-name>.goskope.com/ns#/dashboard`. Grab the `<tenant-name>` part of this URL.
+
```

Comment:
@furi0us333 here's the change I made regarding the tenant configuration. Let me know what you think.

Reply:
Should be good enough!